README.md

```diff
@@ -1 +1,64 @@
-# nvim-package-specification
\ No newline at end of file
+# `plugin.json`
+
+The neovim package specification supports a single, top-level package metadata file. This file can be *either* 'plugin.lua' or 'plugin.json'. The file is a superset of the Rockspec File Format and can contain the following fields
+
+* `package` : String, the name of the package
+
+* `version` : String, the version of the package. Should obey semantic versioning conventions and be prefixed with a `v`, for example, `v0.1.0`
+
+* `specification_version` : String, the current specification version. (v0.1.0) at this time.
+
+
+* `source` : Table, the source is a table that contains a `url` field, which points to either the git commit of the current version of the package, or the source tarball.
+
+* `description` : Table, the description is a table that includes the following nested fields:
+	* `summary` : String, a short description of the package, typically less than 100 character long.
+	* `detailed` : String, a long-form description of the package, this should convey the package's principal functionality to the user without being as detailed as the package readme.
+	* `homepage` : This is the homepage of the package, which in most cases will be the GitHub URL.
+	* `License` : This is the License type of the package.
+
+* `dependencies`
+** TBD
+
+# Example
+
+```lua
+package = "lspconfig"
+version = "v0.1.2"
+specification_version = "v0.1.0"
```

Comment:
`v` in `v0.1.0` seems kinda pointless.

Reply:
I mostly did this because default GH releases add the `v`, idk @lewis6991 @wbthomason wdyt?

Reply:
I'd drop it, but implementations should probably  handle it.